src/lib/sfge/SfgeWrapper.ts

```diff
@@ -33,6 +33,8 @@ type SfgeWrapperOptions = {
 	spinnerManager: SpinnerManager;
 	jvmArgs?: string;
 	pathExpLimit?: number;
+	enablecaching?: boolean;
+	cachepath?: string;
```

Comment:
These properties don't need to be added to both the `SfgeWrapperOptions` and `SfgeExecuteOptions` types. They should only be necessary on the `ExecuteOptions` one.